recipes/wub/meta.yaml

```diff
@@ -6,7 +6,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 1
+  number: 2
```

Comment:
`htslib` in not pinned

Reply:
maybe this is needed, I will leave this to the maintainer

Reply:
We just don't need to bump the build number here.